### PR TITLE
Sort deployments on the overview

### LINKED ISSUE #1555: Upstream deployments not sorted by name on overview
Other resources are sorted, by upstream deployments are not. See https://github.com/openshift/origin-web-console/pull/1553#pullrequestreview-38813433

cc @jwforres

### PR DESCRIPTION
**Pull request by spadgett**
Fixes #1555

### CODE REVIEW

File: app/scripts/controllers/newOverview.js [hunk 1]
```diff
@@ -835,7 +836,7 @@ function OverviewController($scope,
     var toUpdate = [
       overview.deploymentConfigs,
       overview.vanillaReplicationControllers,
-      overview.deploymentsByUID,
+      overview.deployments,
```
Comment (jwforres) [inline app/scripts/controllers/newOverview.js]: were you referencing deploymentsByUID in the view?

Comment (spadgett) [inline app/scripts/controllers/newOverview.js]: Yes, and I had fixed it... I must not have added the file to the commit. Will fix.

**Comment by spadgett** [comment]
@jwforres thanks for catching that, updated

**Comment by spadgett** [comment]
Rebased

**Comment by jwforres** [comment]
[merge]

**Pull request merged**